### PR TITLE
Matlab odometry

### PR DESCRIPTION
**Pull request by agalbachicar**
Adds some matlab scripts to compute open loop trajectories. Only diff drive kinematics are evaluated.

### CODE REVIEW

File: matlab/compute_trajectories.m [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+% Cleans the workspace
+clear all;
+close all;
+clc;
+
+fc = 1;
+% Sampling rate + time vector.
+fs = 20; 
+dt = 1/fs;
+t = 0:0.01:10;
```
Comment (hidmic) [inline matlab/compute_trajectories.m]: @agalbachicar shouldn't `dt` and the range step here match?

File: matlab/odometry_step.m [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+if w ~= 0
+    ICC = [pose_i(1)-R*sin(pose_i(3)); pose_i(2)+R*cos(pose_i(3))];
+    pose_f = ...
+        [cos(phase) -sin(phase) 0; sin(phase) cos(phase) 0; 0 0 1] * ...
+        [pose_i(1)-ICC(1); pose_i(2)-ICC(2); pose_i(3)] + ...
+        [ICC(1); ICC(2); phase];
+else
+   pose_f = [pose_i(1)-v*dt*sin(pose_i(3)); ...
+             pose_i(2)+v*dt*cos(pose_i(3)); ...
+             pose_i(3)];
```
Comment (hidmic) [inline matlab/odometry_step.m]: @agalbachicar hmm, there's something weird here. Maybe I'm a bit burned, but shouldn't this be `[x, y, theta] + R2(theta) * [v*dt, 0, 0]`, with `R2(theta)` being the regular planar 2D rotation matrix?

**Review by hidmic — CHANGES_REQUESTED** [review]
First pass, I'll review the math better when I'm less burned.

File: matlab/odometry_step.m (hunk 2 above)
Comment (agalbachicar) [reply matlab/odometry_step.m]: Good catch! Thanks.

**Comment by agalbachicar** [comment]
Good catches so far! Thanks and comments were addressed. Feel free to do another review whenever you can.

**Review by hidmic — APPROVED** [review]

**Pull request merged**